src/components/WalletConnectWrapper/WalletConnecWrapper.component.tsx

```diff
@@ -0,0 +1,30 @@
+import Typography from '@mui/material/Typography';
+
+import { useActiveWeb3React } from '../../hooks/useActiveWeb3React';
+import { WalletConnectInfoProps } from './WalletConnectWrapper.types';
+
+export const WalletConnectWrapper = ({
```

Comment:
For me the name of this component is unintuitive. As I understand it allows us to force user connecting to a specific chain (RSK here). Name sounds as it provides some crutial functionality, when it basically just checks

Reply:
I agree. What do you think about `WalletConnectionChecker`?